day03/deploy_django_app.sh

```
@@ -1,75 +1,75 @@
-code_clone() {
-    echo "Cloning the Django app..."
-    if [ -d "django-notes-app" ]; then
-        echo "The code directory already exists. Skipping clone."
-    else
-        git clone https://github.com/LondheShubham153/django-notes-app.git || {
-            echo "Failed to clone the code."
-            return 1
-        }
-    fi
+
+<< task
```

Shellcheck (0.11.0)

[warning] This redirection doesn't have a command. Move to its command (or use 'true' as no-op).

(SC2188)